internal/transform/operation.go

```diff
@@ -13,10 +14,12 @@ import (
 	"github.com/stellar/go/xdr"
 )
 
+type Claimants []Claimant
+
 // TransformOperation converts an operation from the history archive ingestion system into a form suitable for BigQuery
 func TransformOperation(operation xdr.Operation, operationIndex int32, transaction ingest.LedgerTransaction, ledgerSeq int32) (OperationOutput, error) {
 	outputTransactionID := toid.New(ledgerSeq, int32(transaction.Index), 0).ToInt64()
-	outputOperationID := toid.New(ledgerSeq, int32(transaction.Index), operationIndex).ToInt64()
+	outputOperationID := toid.New(ledgerSeq, int32(transaction.Index), operationIndex+1).ToInt64() //operationIndex needs +1 increment
```

Comment:
I do not have a great reason, I just noticed that our operation ids were off by 1 during data validation. Horizon increments by 1 [here](https://github.com/stellar/go/blob/ab0780781eece93b937043273bf090bf35dfc0f5/services/horizon/internal/ingest/processors/operations_processor.go#L92)....maybe just because the id is 1 indexed instead of 0 indexed?

Reply:
ya taking a quick look at the `ingest` code I think you're right. Maybe comment could be `// operationIndex needs +1 increment to be in sync with ingest package`, or something like that 🤔